doc/source/dev/gitwash/git_links.inc

```diff
@@ -13,20 +13,17 @@
 .. _github: https://github.com/numpy/numpy
 .. _github help: https://help.github.com
 .. _`install it`: https://git-scm.com/downloads
-.. _subversion: http://subversion.tigris.org/
-.. _git cheat sheet: http://cheat.errtheblog.com/s/git
```

Comment:
I am unable to identify this specific cheat sheet. There are dozens of Git cheat sheets floating around, do we really need to suggest one?

Reply:
I would say, if we suggest one, we that one close to the official manual/site/people, rather than pick a random blogpost.

Reply:
The [git Documentation](https://git-scm.com/doc) page links to the [GitHub cheat sheet](https://training.github.com/downloads/github-git-cheat-sheet.pdf), but I don't find it that good/readable.

Reply:
Let it be an upstream problem then? E.g. we link to the git Documentation, and then trust they find the best resources to link to from there. 

Maybe, in addition to the git docs, we could link to software carpentry?

Reply:
This would involve rewriting this page:
https://numpy.org/devdocs/dev/gitwash/git_resources.html

I do not find the git documentation page very useful, and the software carpentry mostly advertises seminars.

I suggest we keep this page as is for now.